src/Helpers/Emojify.php

```diff
@@ -158,9 +158,13 @@ protected function replace($line, $replace, $toWord = false, $delimiter = ':')
      */
     protected function wordToEmojiReplace($line, $replace, $delimiter)
     {
-        foreach ($replace as $key => $value) {
-            $line = str_replace($delimiter.$key.$delimiter, $value, $line);
-        }
+        $pattern = '/' . preg_quote($delimiter, '/') . '([^:]*)' . preg_quote($delimiter, '/') . '/';
```

Comment:
You've forgotten to put here `[^:]` calling `preg_quote($delimiter, '/')` as well, I think.